lib/wordrep.gi

```diff
@@ -468,6 +468,12 @@ InstallMethod( \*,
     [ Is8BitsAssocWord, Is8BitsAssocWord ], 0,
     8Bits_Product );
 
+InstallOtherMethod( \/,
+    "for two 8 bits assoc. words",
+    IsIdenticalObj,
+    [ Is8BitsAssocWord, Is8BitsAssocWord ], 0,
+    8Bits_Quotient );
```

Comment:
This method would be applicable also to elements of, for example, `FreeMonoid( IsSyllableWordsFamily, 2 )`,
with the effect that corrupted objects arise.
If `IsMultiplicativeElementWithInverse` is required for the second argument then this cannot happen,
and one can use `InstallMethod` instead of `InstallOtherMethod`.